package.json

```diff
@@ -42,7 +42,7 @@
     "cheerio": "^0.22.0",
     "eslint": "^7.0.0",
     "eslint-config-hexo": "^4.1.0",
-    "hexo": "^4.0.0",
+    "hexo": "hexojs/hexo",
```

Comment:
Pending release of Hexo 5.

Reply:
@SukkaW okay with `"hexo": "hexojs/hexo"`, instead of `"^5.0.0"`?

Reply:
@curbengh We can change this back to `^5.0.0` when release a newer version.

Reply:
I'll just wait then.

Reply:
dependabot PR: https://github.com/hexojs/hexo-generator-feed/pull/152